server/blob/blob-api/src/main/java/org/apache/james/blob/api/BlobStore.java

```diff
@@ -43,45 +45,118 @@ interface BlobIdProvider {
 
     Publisher<BlobId> save(BucketName bucketName, byte[] data, StoragePolicy storagePolicy);
 
+    default Publisher<BlobId> save(Bucket bucket, byte[] data, StoragePolicy storagePolicy) {
+        return save(bucket.bucketName(), data, storagePolicy);
+    }
```

Comment:
This makes it harder to use the class: we can not double the number of methods every time we add some parameters.

I'd be happy we make it easy to get a Bucket from a BucketName and sollely depend on bucket